curriculum/challenges/english/25-front-end-development/lecture-working-with-client-side-storage-and-crud-operations/6733ab64775d35f78f5238fe.md

```diff
@@ -12,7 +12,6 @@ Watch the video or read the transcript and answer the questions below.
 
 # --transcript--
```

Comment:
You seem to have forgot to remove the transcript label. Can you please remove it?